debezium-embedded/src/main/java/io/debezium/embedded/async/AsyncEngineSignaler.java

```diff
@@ -6,36 +6,21 @@
 package io.debezium.embedded.async;
 
 import java.util.List;
-import java.util.Optional;
 
-import io.debezium.DebeziumException;
 import io.debezium.engine.DebeziumEngine;
-import io.debezium.engine.source.EngineSourceTask;
 import io.debezium.pipeline.signal.SignalRecord;
-import io.debezium.pipeline.signal.channels.process.InProcessSignalChannel;
+import io.debezium.pipeline.signal.channels.process.SignalChannelWriter;
 
-public class AsyncEngineSignaler implements DebeziumEngine.Signaler<SignalRecord> {
-    private AsyncEmbeddedEngine<?> engine;
-    private List<InProcessSignalChannel> channels;
+public class AsyncEngineSignaler implements DebeziumEngine.Signaler {
```

Comment:
this could be now named `EmbeddedSignaler` and moved to the parent package as the implementation itself is not tied to async engine anymore

Reply:
Do you plan to do that change?

Reply:
Renamed